Makefile

```diff
@@ -31,7 +31,7 @@ ifeq ($(BUILD_TYPE),debug)
 BUILDFLAGS += -gcflags "-N -l"
 endif
 
-RELEASE_VER := 2.7.0
```

Comment:
Lets change this to 23.2.0  and the branch name should be 23.2 - This was the standard we have been following in the past